src/DoctrineMongoODMModule/Module.php

```diff
@@ -42,8 +41,7 @@
  * @since   0.1.0
  * @author  Tim Roediger <superdweebie@gmail.com>
  */
-class Module implements
-    BootstrapListenerInterface,
+final class Module implements
```

Comment:
I could revert this but there is no use-case for extending the `Module` class, and doing so would be incorrect. I understand that from a purist's point-of-view this would be a BC break but in practice it is not.